packages/migrate/sources/jekyll.js

```diff
@@ -0,0 +1,194 @@
+    }
+
+    if (scrapedData.twitter_image) {
+        scrapedData.twitter_image = scrapedData.twitter_image.replace(/(?:-\d{2,4}x\d{2,4})(.\w+)$/gi, '$1');
+    }
+
+    if (options.featureImage === 'og:image' && scrapedData.og_image) {
+        data.feature_image = scrapedData.og_image;
+    }
+
+    return scrapedData;
+};
+
+/**
+ * getTasks: Steps to Migrate from Jekyll
+ *
+ * Wiring of the steps to migrate from Jekyll.
+ *
+ * @param {String} pathToZip
+ * @param {Object} options
+ */
+module.exports.getTaskRunner = (pathToZip, options) => {
+    let tasks = [
+        {
+            title: 'Initialising Workspace',
+            task: (ctx, task) => {
+                ctx.options = options;
+
+                // 0. Prep a file cache, scrapers, etc, to prepare for the work we are about to do.
+                ctx.fileCache = new fsUtils.FileCache(pathToZip);
+                ctx.jekyllScraper = new MgWebScraper(ctx.fileCache, scrapeConfig, postProcessor);
+                ctx.imageScraper = new MgImageScraper(ctx.fileCache);
+                ctx.linkFixer = new MgLinkFixer();
+
+                task.output = `Workspace initialised at ${ctx.fileCache.cacheDir}`;
+            }
+        },
+        {
+            title: 'Read Jekyll export zip',
+            task: async (ctx) => {
+                // 1. Read the zip file
+                try {
+                    ctx.result = jekyllIngest(pathToZip, options);
+                    await ctx.fileCache.writeTmpFile(ctx.result, 'jekyll-export-data.json');
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Fetch missing data via WebScraper',
+            task: (ctx) => {
+                // 2. Pass the results through the web scraper to get any missing data
+                let tasks = ctx.jekyllScraper.hydrate(ctx); // eslint-disable-line no-shadow
+                return makeTaskRunner(tasks, options);
+            },
+            skip: () => ['all', 'web'].indexOf(options.scrape) < 0
+        },
+        {
+            title: 'Build Link Map',
+            task: async (ctx) => {
+                // 3. Create a map of all known links for use later
+                try {
+                    ctx.linkFixer.buildMap(ctx);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Format data as Ghost JSON',
+            task: (ctx) => {
+                // 4. Format the data as a valid Ghost JSON file
+                try {
+                    ctx.result = mgJSON.toGhostJSON(ctx.result, ctx.options);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Fetch images via ImageSraper',
```

Comment:
s/Sraper/Scraper/

Reply:
This was fixed as of https://github.com/TryGhost/migrate/commit/8e50ea12527b8623a9fb6745e8db71431d4db4e8